DEVELOPMENT.md

```diff
@@ -0,0 +1,80 @@
+# Developing
+
+## Setting up a development environment
+
+### Setup a GitHub account accessible via SSH
+
+GitHub is used for project Source Code Management (SCM) using the SSH protocol for authentication.
+
+1. Create [a GitHub account](https://github.com/join) if you do not already have one.
+1. Setup
+[GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
+
+### Install tools
+
+You must install these tools:
+
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+1. [`java`](https://www.oracle.com/java/technologies/downloads/): The language this SDK is built in.
```

Comment:
Should this be sequential? I see in other parts of the guide you use all `1.` and was curious if thats a standard you have seen or if this will be more easily read by end users? Thanks!

Reply:
`1.` is used in markdown to mark a numbered bullet list. The markdown rendering engine replaces the ones with a numbered list.
The advantage of using `1.` everywhere is that one doesn't have to renumber all items when adding something new in the middle

Reply:
Oh this is a neat feature, I'm a novice user of markdown (using the basic design capabilities to create readme) and will start using this now!